src/cli/cli.cpp

```diff
@@ -4233,6 +4295,20 @@ template <> otError Interpreter::Process<Cmd("mode")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli multiradio
+ * @code
+ * multiradio
+ * [15.4, TREL]
+ * Done
+ * @endcode
+ * @par
+ * Get the list of supported radio links by the device.
+ * @par
+ * This command is always available, even when only a single radio is supported by the device.
+ * @par api_copy
+ * #otMultiRadioGetNeighborInfo
```

Comment:
We shouldn't link this API here, it's for the neighbor info that requires OPENTHREAD_CONFIG_MULTI_RADIO